snuba/api.py

```diff
@@ -374,14 +374,14 @@ def ensure_table_exists(dataset, force=False):
             dataset.get_schema().get_local_table_definition()
         )
 
-        migrate.run(clickhouse_rw, dataset.get_schema().get_local_table_name())
```

Comment:
It looks like [`migrate.run` isn't specific to the local table](https://github.com/getsentry/snuba/pull/323/files#diff-701dc91b072bf9b1fa9d46a6e02ce575R11). That seems OK for this specific method (and the other CLI call site) — should `migrate.run` also perform the local check? Or is it OK if that function is called in the distributed mode?

Reply:
Actually no, we should make migrate specific to the local table instead. It cannot act on a distributed table but it can safely work on a local table. even in distributed mode (not that we are doing it today).